.github/workflows/ci.yml

```diff
@@ -149,6 +149,8 @@ jobs:
           TOX_SKIP_ENV: ${{ matrix.tox_skip_env }}
       - name: Publish coverage data to codecov
         uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
So this may still be flaky, in my experience. In many projects, I've migrated to using a token hard-coded in the `.codecov.yml` config, in the open. It's not a very sensitive value, after all. But it does make uploads from PRs much more stable.

Reply:
Well, we're stuck at the same problem.  I don't think that I have sufficient privs at codecov.io to update/access the appropriate token.  (Once we get the right one, I would assume the GitHub secrets would work as designed to propagate that value?)

Anyway, any current issues I think are due to not having the correct token.

(Getting this working is not a high priority, personally.)